data/convert_tau_sofa.py

```diff
@@ -0,0 +1,16 @@
+import sofa_utils
+import os
+
+tau_db_dir = '/scratch/ci411/SRIR_DATASETS/TAU_SRIR_DB/TAU-SRIR_DB'
+sofa_db_dir = '/scratch/ci411/SRIR_DATASETS/TAU_SRIR_DB_SOFA'
+db_name = "TAU-SRIR-DB-SOFA"
```

Comment:
When I first ran this I got an error:
  
  `PermissionError: [Errno 13] Permission denied: 'TAU_DB/TAU_SRIR_DB_SOFA/foa/bomb_shelter.sofa`
  
  which was solved by `mkdir -p TAU_DB/TAU_SRIR_DB_SOFA/foa`. Let's include this step as python code in this preprocessing step.

Reply:
Added in commit `[a08e28c](https://github.com/iranroman/SELD-data-generator/pull/20/commits/a08e28c023571c31312c27f2946e698e06585d88)`